include/LightningJSON/LightningJSON.hpp

```diff
@@ -442,91 +458,96 @@ namespace LightningJSON
+		JSONObject& Insert(char const* const name, unsigned short value) { return Insert(std::string_view(name, strlen(name)), (unsigned long long)(value)); }
+		JSONObject& Insert(char const* const name, unsigned int value) { return Insert(std::string_view(name, strlen(name)), (unsigned long long)(value)); }
+		JSONObject& Insert(char const* const name, unsigned long value) { return Insert(std::string_view(name, strlen(name)), (unsigned long long)(value)); }
+		JSONObject& Insert(char const* const name, unsigned long long value) { return Insert(std::string_view(name, strlen(name)), value); }
+		JSONObject& Insert(char const* const name, long long value) { return Insert(std::string_view(name, strlen(name)), value); }
+		JSONObject& Insert(char const* const name, float value) { return Insert(std::string_view(name, strlen(name)), (long double)(value)); }
+		JSONObject& Insert(char const* const name, double value) { return Insert(std::string_view(name, strlen(name)), (long double)(value)); }
+		JSONObject& Insert(char const* const name, long double value) { return Insert(std::string_view(name, strlen(name)), value); }
+		JSONObject& Insert(char const* const name, bool value) { return Insert(std::string_view(name, strlen(name)), value); }
+		JSONObject& Insert(char const* const name, const char* const value) { return Insert(std::string_view(name, strlen(name)), value); }
+		JSONObject& Insert(char const* const name, const char* const value, size_t length) { return Insert(std::string_view(name, strlen(name)), value, length); }
+		JSONObject& Insert(char const* const name, std::string const& value) { return Insert(std::string_view(name, strlen(name)), value); }
+		JSONObject& Insert(char const* const name, std::string_view const& value) { return Insert(std::string_view(name, strlen(name)), value); }
 
 		static JSONObject Array()
 		{
-			return JSONObject(JSONType::Array, string_view());
+			return JSONObject(JSONType::Array, std::string_view());
 		}
 
 		static JSONObject Object()
 		{
-			return JSONObject(JSONType::Object, string_view());
+			return JSONObject(JSONType::Object, std::string_view());
 		}
 
 		static JSONObject String(char const* const data)
 		{
-			return JSONObject(JSONType::String, string_view(data));
+			return JSONObject(JSONType::String, std::string_view(data));
 		}
 
 		static JSONObject String(char const* const data, size_t length)
 		{
-			return JSONObject(JSONType::String, string_view(data, length));
+			return JSONObject(JSONType::String, std::string_view(data, length));
+		}
+
+		static JSONObject String(std::string const& data)
+		{
+			return JSONObject(JSONType::String, std::string_view(data.data(), data.length()));
 		}
 
-		static JSONObject String(string_view const& data)
+		static JSONObject String(std::string_view const& data)
 		{
-			return JSONObject(JSONType::String, data);
+			return JSONObject(JSONType::String, std::string_view(data.data(), data.length()));
```

Comment:
Curious why you're creating a new string view when you already have a string view?

Reply:
It was probably left over from the whole `StringView` thing